### PR TITLE
Adds Tutorial for Remote Staking via XCM

### PR DESCRIPTION
**Pull request by themacexpert**
### Description

Adds a tutorial for Remote Staking via XCM. 

### Checklist

- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [ ] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ] If this page requires a disclaimer, I have added one

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

File: tutorials/remote-staking-xcm.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+2. Our XCM Message is going to have 3 distinct XCM instructions, so press the first **Add Item** button 3 times. 
+3. Below the first XCM Instruction of `Withdraw Asset`, we need to add the asset we’re going to withdraw here, so press the **Add Item** button below `Withdraw Asset` once. 
+
+![Preparing the structure of the XCM message](/images/tutorials/remote-staking-via-xcm/xcm-stake-7.png)
+
+### Assembling the Contents of the XCM Message
+
+Now we’re ready for the fun part! To construct the XCM message that will remotely stake funds on the Moonbase Alpha parachain, take the following steps:
+
+1. Double check that `V2` is selected as the XCM version
```
Comment (albertov19) [inline tutorials/remote-staking-xcm.md:91]: Normally I would collapse a lot of the multilocation or multiple settings into a single dropdown with just 1 number, check here: https://docs.moonbeam.network/builders/xcm/remote-evm-calls/#build-xcm-remote-evm

File: tutorials/remote-staking-xcm.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
+---
+title: Remote Staking via XCM
```
Comment (albertov19) [inline tutorials/remote-staking-xcm.md:2]: Maybe in these tutorials we can add something like the date an the name of the guy who wrote it in italics (cc @eshaben not sure what you think. Similar to what is used in the Blogpost section of the website

`December XX, YYYY | by Kevin`

File: tutorials/remote-staking-xcm.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+description: In this guide, we'll be leveraging remote execution to remotely stake GLMR on Moonbeam using a series of XCM instructions.
+---
+
+# Remote Staking via XCM
+
+![Banner Image](/images/tutorials/remote-staking-via-xcm/remote-staking-via-xcm-banner.png)
+
+## Introduction {: #introduction } 
+In this tutorial, we’ll stake DEV tokens remotely by sending XCM instructions from an account on the Moonbase Relay Chain (equivalent to the Polkadot Relay Chain). This tutorial assumes a basic familiarity with [XCM](/builders/xcm/overview/){target=_blank} and [Remote Execution via XCM](builders/xcm/xcm-transactor/){target=_blank}. You don’t have to be an expert on these topics but you may find it helpful to have some XCM knowledge as background.
```
Comment (albertov19) [inline tutorials/remote-staking-xcm.md:11]: I guess we can use 2nd person because it is a blogpost style article? Or should we keep it like other tutorials? (cc @eshaben )

File: tutorials/remote-staking-xcm.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+---
+
+# Remote Staking via XCM
+
+![Banner Image](/images/tutorials/remote-staking-via-xcm/remote-staking-via-xcm-banner.png)
+
+## Introduction {: #introduction } 
+In this tutorial, we’ll stake DEV tokens remotely by sending XCM instructions from an account on the Moonbase Relay Chain (equivalent to the Polkadot Relay Chain). This tutorial assumes a basic familiarity with [XCM](/builders/xcm/overview/){target=_blank} and [Remote Execution via XCM](builders/xcm/xcm-transactor/){target=_blank}. You don’t have to be an expert on these topics but you may find it helpful to have some XCM knowledge as background. 
+
+There are actually two possible approaches for staking on Moonbeam remotely via XCM. We could send a remote EVM call that calls the [staking precompile](builders/pallets-precompiles/precompiles/staking/){target=_blank}, or we could use XCM to call the [parachainStaking pallet](builders/pallets-precompiles/pallets/staking/){target=_blank} directly without interacting with the EVM. For this tutorial, we’ll be taking the latter approach and interacting with the parachainStaking pallet directly.
```
Comment (albertov19) [inline tutorials/remote-staking-xcm.md:13]: Use the standards we normally apply here:

```
- All UI elements should be in bold. This is done in some places, but other places it looks like it was forgotten
- Typically on the docs site, code-related variables, functions, addresses, and file names are typically in back ticks but for blog posts we italicize these items (unless they are a UI element, then they should be bold)
- Staking Precompile doesn't need to be capitalized
- WEI should be Wei
```

So I guess it should be `parachainStaking`

File: tutorials/remote-staking-xcm.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+Copy the account of your existing or newly created account on the [Moonbase Relay Chain](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffrag-moonbase-relay-rpc-ws.g.moonbase.moonbeam.network#/accounts){target=_blank}. You're going to need it to calculate the corresponding derivative account. A derivative account is a special type of account that’s keyless (the private key is unknown). Transactions from a derivative account can be initiated only via valid XCM instructions from the corresponding account on the relay chain. In other words, you are the only one who can initiate transactions on your derivative account - and if you lose access to your Moonbase Relay account, you’ll also lose access to your derivative account.
```
Comment (albertov19) [inline tutorials/remote-staking-xcm.md:20]:
```suggestion
Copy the account of your existing or newly created account on the [Moonbase Relay Chain](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffrag-moonbase-relay-rpc-ws.g.moonbase.moonbeam.network#/accounts){target=_blank}. You're going to need it to calculate the corresponding multilocation derivative account. A multilocation derivative account is a special type of account that’s keyless (the private key is unknown). Transactions from a multilocation derivative account can be initiated only via valid XCM instructions from the corresponding account on the relay chain. In other words, you are the only one who can initiate transactions on your multilocation derivative account - and if you lose access to your Moonbase Relay account, you’ll also lose access to your  multilocation derivative account. 
```

Comment (albertov19) [reply tutorials/remote-staking-xcm.md:20]: We refer to it as multilocation derivative account. Fixing this I realized that the word _multilocation derivative_ account is repeated a lot here, so maybe rephrase it a bit

Comment (albertov19) [reply tutorials/remote-staking-xcm.md:20]: Please fix this in the rest of the document (`derivative` for `multilocation derivative`)

File: tutorials/remote-staking-xcm.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+![Moonbeam Network Apps Dashboard](/images/tutorials/remote-staking-via-xcm/xcm-stake-1.png)
+
+Then, head to [Moonbase Alpha Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.testnet.moonbeam.network#/accounts){target=_blank}. In order to see the correct menus here, you’ll need to have at least one account accessible in Polkadot.js Apps. If you don’t, create one now. Then, head to the **Developer** tab and Press **Extrinsics**. 
+
+![Moonbase Alpha Polkadot JS Apps Home](/images/tutorials/remote-staking-via-xcm/xcm-stake-2.png)
+
+In the following steps you will be preparing a transaction, but you’ll need to refrain from submitting the transaction here in order to complete this tutorial in its entirety. We’ll take the resulting encoded call data from preparing this staking operation, and send it via XCM from the relay chain in a later step. From the **Extrinsics** page, take the following steps:
```
Comment (albertov19) [inline tutorials/remote-staking-xcm.md:42]: I feel that we should also provide short code snippets on how to construct these call datas using Polkadot/API, let's discuss this in our team call

**Review by albertov19 — CHANGES_REQUESTED** [review]
Looks good! Left some comments

File: tutorials/remote-staking-xcm.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+
+## Preparing to Stake on Moonbase Alpha
+First and foremost, you’ll need the address of the collator you want to delegate to. To locate it, head to the [Moonbase Alpha Staking dApp](https://apps.moonbeam.network/moonbase-alpha/staking){target=_blank} in a second window. Ensure you’re on the correct network, then press **Select a Collator**. Next to your desired collator, press the **Copy** icon. You’ll also need to make a note of the number of delegations your collator has. The [PS-31 collator](https://moonbase.subscan.io/account/0x3A7D3048F3CB0391bb44B518e5729f07bCc7A45D){target=_blank} shown below has `60` delegations at the time of writing. 
+
+![Moonbeam Network Apps Dashboard](/images/tutorials/remote-staking-via-xcm/xcm-stake-1.png)
+
+Then, head to [Moonbase Alpha Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.testnet.moonbeam.network#/accounts){target=_blank}. In order to see the correct menus here, you’ll need to have at least one account accessible in Polkadot.js Apps. If you don’t, create one now. Then, head to the **Developer** tab and Press **Extrinsics**.
```
Comment (eshaben) [inline tutorials/remote-staking-xcm.md:38]: maybe explicitly state the **Extrinsic** menu instead of correct menus

File: tutorials/remote-staking-xcm.md [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+![Moonbeam Network Apps Dashboard](/images/tutorials/remote-staking-via-xcm/xcm-stake-1.png)
+
+Then, head to [Moonbase Alpha Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.testnet.moonbeam.network#/accounts){target=_blank}. In order to see the correct menus here, you’ll need to have at least one account accessible in Polkadot.js Apps. If you don’t, create one now. Then, head to the **Developer** tab and Press **Extrinsics**. 
+
+![Moonbase Alpha Polkadot JS Apps Home](/images/tutorials/remote-staking-via-xcm/xcm-stake-2.png)
+
+In the following steps you will be preparing a transaction, but you’ll need to refrain from submitting the transaction here in order to complete this tutorial in its entirety. We’ll take the resulting encoded call data from preparing this staking operation, and send it via XCM from the relay chain in a later step. From the **Extrinsics** page, take the following steps:
+
+1. Select the `parachainStaking` Pallet
+2. Select the `delegate` function
```
Comment (eshaben) [inline tutorials/remote-staking-xcm.md:45]: these are UI elements, so they should be bold

File: tutorials/remote-staking-xcm.md [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+3. Paste in your selected collator’s address
+4. Paste your desired stake amount in Wei. In the below example 1 DEV or `1000000000000000000` Wei is specified. You can find a unit converter here on [Moonscan](https://moonscan.io/unitconverter){target=_blank}.
```
Comment (eshaben) [inline tutorials/remote-staking-xcm.md:47]: this is good as is in back ticks as it represents a value that the user needs to enter 👍

File: tutorials/remote-staking-xcm.md [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+5. Enter the collator’s number of existing delegations (this can be found next to the collator’s name / address on the [Moonbase Alpha Staking dApp](https://apps.moonbeam.network/moonbase-alpha/staking){target=_blank}).
```
Comment (eshaben) [inline tutorials/remote-staking-xcm.md:48]: no need for punctuation at the end of steps or lists

File: tutorials/remote-staking-xcm.md [hunk 11]
```diff
@@ -0,0 +1,118 @@
...
+!!! note
+    Astute readers may notice the selected account below is named “Academy.” It does not matter which account you have selected in Moonbase Alpha Polkadot.js Apps. This is because you're not submitting the prepared transaction, only copying the encoded call data, which does not contain a reference to the sending account. 
+
+![Moonbase Alpha Polkadot JS Apps Extrinsics Page](/images/tutorials/remote-staking-via-xcm/xcm-stake-3.png)
+
+## Sending the XCM Instructions from the Moonbase Relay chain
+
+In another tab, head to [Moonbase Relay Polkadot.Js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffrag-moonbase-relay-rpc-ws.g.moonbase.moonbeam.network#/extrinsics){target=_blank}. Click on the **Developer** tab and press **Extrinsics**. 
+
+![Moonbase Relay Polkadot JS Apps Home](/images/tutorials/remote-staking-via-xcm/xcm-stake-4.png)
```
Comment (eshaben) [inline tutorials/remote-staking-xcm.md:61]: there is a red border on the left side and bottom of this image

Comment (eshaben) [reply tutorials/remote-staking-xcm.md:61]: same with xcm-stake-2.png

File: tutorials/remote-staking-xcm.md [hunk 12]
```diff
@@ -0,0 +1,118 @@
...
+
+### Building the Destination Multilocation
```
Comment (eshaben) [inline tutorials/remote-staking-xcm.md:63]: this is a nitpick haha but can you please add a line break between titles and the first sentence of the paragraph

File: tutorials/remote-staking-xcm.md [hunk 13]
```diff
@@ -0,0 +1,118 @@
...
+![Moonbase Alpha Polkadot JS Apps Extrinsics Page](/images/tutorials/remote-staking-via-xcm/xcm-stake-3.png)
+
+## Sending the XCM Instructions from the Moonbase Relay chain
+
+In another tab, head to [Moonbase Relay Polkadot.Js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffrag-moonbase-relay-rpc-ws.g.moonbase.moonbeam.network#/extrinsics){target=_blank}. Click on the **Developer** tab and press **Extrinsics**. 
+
+![Moonbase Relay Polkadot JS Apps Home](/images/tutorials/remote-staking-via-xcm/xcm-stake-4.png)
+
+### Building the Destination Multilocation 
+Let’s get started crafting our XCM message that will transport our remote execution instructions to the Moonbase Alpha Parachain to ultimately stake our desired amount of DEV tokens to a chosen collator. To get started, take the following steps:
```
Comment (eshaben) [inline tutorials/remote-staking-xcm.md:64]: parachain doesn't need to be capitalized

File: tutorials/remote-staking-xcm.md [hunk 14]
```diff
@@ -0,0 +1,118 @@
...
+7. Select `parachain ID` 
+8. Enter `1000`, the parachain ID of Moonbase Alpha 
+
+![Moonbase Relay Polkadot JS Apps Extrinsics Page](/images/tutorials/remote-staking-via-xcm/xcm-stake-5.png)
+
+In the next section, we’ll start assembling the XCM instructions. 
+
+### Preparing the Structure of the XCM Message
+
+1. Select `V2` for XcmVersionedXcm
```
Comment (eshaben) [inline tutorials/remote-staking-xcm.md:81]:
```suggestion
1. Select **V2** for **XcmVersionedXcm**
```

both are UI elements here

File: tutorials/remote-staking-xcm.md [hunk 15]
```diff
@@ -0,0 +1,118 @@
...
+2. Our XCM Message is going to have 3 distinct XCM instructions, so press the first **Add Item** button 3 times. 
+3. Below the first XCM Instruction of `Withdraw Asset`, we need to add the asset we’re going to withdraw here, so press the **Add Item** button below `Withdraw Asset` once.
```
Comment (eshaben) [inline tutorials/remote-staking-xcm.md:83]: no spaces in the UI so it should be **WithdrawAsset**

File: tutorials/remote-staking-xcm.md [hunk 16]
```diff
@@ -0,0 +1,118 @@
...
+18. Select `Unlimited` for the weightLimit
+19. For our 3rd and most crucial XCM instruction, choose the `Transact` instruction
+20. Specify an origin of the `SovereignAccount`
+21. For requireWeightAtMost, specify `40000000000`. This is likely more than strictly necessary, but it is fine to overestimate
+22. Finally, paste the encoded call data of the staking operation that you prepared on Moonbase Alpha in Step 7 of the first section of this tutorial
+23. Press **Submit** and enter your password in the resulting popup to confirm the transaction
+
+![Assembling the complete XCM message](/images/tutorials/remote-staking-via-xcm/xcm-stake-6.png)
+
+And that’s it! To verify that your delegation was successful, you can visit [Subscan](https://moonbase.subscan.io/){target=_blank} to check your staking balance. Be advised that it may take a few moments before your staking balance is visible on Subscan. Additionally, be aware that you will not be able to see this staking operation on Moonscan, because we initiated the delegation action directly via the parachainStaking pallet (on the substrate side) rather than through the staking precompile (on the EVM).
```
Comment (eshaben) [inline tutorials/remote-staking-xcm.md:117]:
```suggestion
And that’s it! To verify that your delegation was successful, you can visit [Subscan](https://moonbase.subscan.io/){target=_blank} to check your staking balance. Be advised that it may take a few moments before your staking balance is visible on Subscan. Additionally, be aware that you will not be able to see this staking operation on Moonscan, because we initiated the delegation action directly via the parachain staking pallet (on the substrate side) rather than through the staking precompile (on the EVM). 
```

**Review by eshaben — CHANGES_REQUESTED** [review]
So exciting! First tutorial!! 🚀 

Main comment is that UI elements should be bolded and values/variables/functions (that aren't UI elements or the user has to manually enter) should be in back ticks

**Comment by eshaben** [comment]
@themacexpert Can you please merge in master whenever you get a chance? It'll fix the broken build

File: tutorials/remote-staking-xcm.md [hunk 17]
```diff
@@ -0,0 +1,118 @@
...
+To generate the derivative account, first clone Alberto’s [xcmTools repo](https://github.com/albertov19/xcmTools){target=_blank}. Run `yarn` to install the necessary packages and then run:
+
+
+    ts-node calculateMultilocationDerivative.ts \
+     --w wss://wss.api.moonbase.moonbeam.network \
+     --a YOUR-MOONBASE-RELAY-ACCOUNT-HERE \
+     --n 0x57657374656e64
+
+
+The `-w` flag corresponds to the endpoint we’re using to fetch this information. The `-n` flag corresponds to the encoded form of “westend”, the name of the relay chain that Moonbase Relay is based on. The script will return 32-byte and 20-byte addresses. We’re interested in the ethereum-style account - the 20-byte one. Feel free to look up your derivative account on [Moonscan](https://moonbase.moonscan.io/){target=_blank}. You’ll note that this account is empty. You’ll now need to fund this account with at least 2 DEV.
```
[the remark this answers is not in the document]
Comment (themacexpert) [reply tutorials/remote-staking-xcm.md:36]: I modified to 1.1 DEV as that is the minimum needed on the derivative account but they'll still need a bit more to make the xcUNIT swap. Will include a bit about reaching out on Discord for more tokens.

**Comment by themacexpert** [comment]
Thank you so much for the feedback @albertov19 @eshaben, I believe I've fixed everything over at https://github.com/PureStake/moonbeam-docs/pull/487
